types/jquery-lazyload/index.d.ts

```diff
@@ -15,7 +15,7 @@ declare namespace JQueryLazyLoad {
         container?: JQuery;
         data_attribute?: string;
         skip_invisible?: boolean;
-        appear?: null;
+        appear?: (elementsLeft: number, options: Options) => void;
```

Comment:
Should `null` still be allowed. the check in https://github.com/tuupola/jquery_lazyload/blob/218e50eb4999fe59ac94b939a65c8c988d1d420b/jquery.lazyload.js#L101, allows `null`.